web/onos-gui/src/app/onos-config/change-value.util.ts

```diff
@@ -35,24 +35,22 @@ export class ChangeValueUtil {
             return [];
         }
         let valueStrings: string[] = [];
+        const dataIsLittleEndian = true;
         switch (value.valueType) {
             case ChangeValueType.BOOL:
                 valueStrings = [value[0] ? 'true' : 'false'];
                 break;
             case ChangeValueType.INT:
-                // TODO fix this for Int
                 const view1 = new DataView(value.value.buffer, 0, 8);
-                valueStrings = [String(view1.getInt32(0))];
+                valueStrings = [String(view1.getInt32(0, dataIsLittleEndian))];
                 break;
             case ChangeValueType.UINT:
-                // TODO fix this for UInt
                 const view2 = new DataView(value.value.buffer, 0, 8);
-                valueStrings = [String(view2.getUint32(0))];
+                valueStrings = [String(view2.getUint32(0, dataIsLittleEndian))];
```

Comment:
I tried an end to end test with the following config:
```
gnmi_cli -address onos-config:5150 -set \
    -proto "update: <path: <target: 'test1', elem: <name: 'cont1a'> elem: <name: 'cont2a'> elem: <name: 'leaf2a'>> val: <uint_val: 12>> extension: <registered_ext: <id: 100, msg:'added_a_testdevice'>> extension: <registered_ext: <id: 101, msg:'1.0.0'>> extension: <registered_ext: <id: 102, msg:'TestDevice'>>" \
    -timeout 5s -alsologtostderr \
    -client_crt /etc/ssl/certs/client1.crt \
    -client_key /etc/ssl/certs/client1.key \
    -ca_crt /etc/ssl/certs/onfca.crt
```
but it shows **2651327498** in the onos-gui view.

I don't know how this can be - is the browser compiling it differently?

I added a specific test that takes the same array.
In the test I get 
```
'UINT value', Uint8Array{0: 12, 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 0, 7: 0}, 'converted to', ['12']
```
In the browser I get:
```
UINT value Uint8Array(8) [12, 0, 0, 0, 0, 0, 0, 0] 3 converted ["2651327498"]
```